armeria/src/main/java/brave/example/HttpTracingFactory.java

```diff
@@ -0,0 +1,86 @@
+package brave.example;
+
+import brave.Tracing;
+import brave.baggage.BaggageField;
+import brave.baggage.BaggagePropagation;
+import brave.baggage.BaggagePropagationConfig;
+import brave.baggage.CorrelationScopeConfig;
+import brave.context.slf4j.MDCScopeDecorator;
+import brave.http.HttpTracing;
+import brave.propagation.B3Propagation;
+import brave.propagation.CurrentTraceContext;
+import brave.propagation.CurrentTraceContext.ScopeDecorator;
+import brave.propagation.Propagation;
+import com.linecorp.armeria.common.brave.RequestContextCurrentTraceContext;
+import java.io.IOException;
+import java.util.logging.Logger;
+import zipkin2.reporter.Sender;
+import zipkin2.reporter.brave.AsyncZipkinSpanHandler;
+import zipkin2.reporter.urlconnection.URLConnectionSender;
+
+final class HttpTracingFactory {
```

Comment:
main differences here are baggage configuration, SLF4J integration and export of HttpTracing